src/olympia/activity/utils.py

```diff
@@ -195,3 +218,20 @@ def filter_queryset_to_pending_replies(queryset, log_type_ids=NOT_PENDING_IDS):
     if not latest_reply:
         return queryset
     return queryset.filter(created__gt=latest_reply.created)
+
+
+def bounce_mail(message, reason):
+    if (not type(message) is dict or
+            ('From' not in message and 'ReplyTo' not in message)):
```

Comment:
This ends up in a large `if`-statement again, can we do something like

```python
recipient = message.get('From', message.get('ReplyTo'))
if not isinstance(message, dict) and not recipient:
    log.error(...)
    return
```

and we'd be able to reuse `recipient` down below where we're passing it to `recipient_list` and avoid a line-break there too.